### ISSUE TITLE
make install error, BBEdit app bundle ctags dir non-existent

### ISSUE THREAD

**Issue by flips** [ascarter/Go-bbpackage#14]
I get this error:

```
$ make install
mkdir -p ./Go.bbpackage/Contents
cp README.md ./Go.bbpackage/.
cp LICENSE ./Go.bbpackage/.
cp -R ./src/* ./Go.bbpackage/Contents/.
/Applications/BBEdit.app/Contents/Resources/ctags --recurse --extra=+p+q+r --fields=+a+m+n+S --excmd=number --tag-relative=no --sort=no --exclude=.git --exclude="*_test.go" --exclude="**/testdata/**" --exclude="**/internal/**" -f - /usr/local/Cellar/go/1.12/libexec/src | grep -v -E '^(\w+[.])?[^A-Z]\w+\s+\S+\s\d+[;]["]\s+[cfimstv]' > ./gostdlib.tags
/bin/sh: /Applications/BBEdit.app/Contents/Resources/ctags: No such file or directory
make: *** [Go.bbpackage/Contents/Completion Data/Go/Go Standard Library.tags] Error 1
```
So there's no Resources/ctags in the app bundle. (Should I make the dir?) 🤔 
Installing by double-clicking the resulting `Go.bbpackage` seems to work, though.
Not sure if I should copy `gostdlib.tags` somewhere. 😄

**Comment by utdrmac** [comment]
Resolution here? Same issue. BBEdit 12.6.7

**Comment by utdrmac** [comment]
Seems to work with this change:
```
CTAGS_CMD ?= /Applications/BBEdit.app/Contents/Helpers/ctags
```

**Comment by ascarter** [comment]
Resolved with 1.0.0 release
